package.json

```diff
@@ -1,21 +1,24 @@
 {
   "name": "@shiftcommerce/shift-react-components",
-  "version": "0.4.0",
+  "version": "0.5.0",
   "main": "dist/bundle.js",
   "license": "MIT",
   "scripts": {
     "start": "NODE_ENV=production ANALYZE_BUNDLE=true ./node_modules/webpack/bin/webpack.js --watch --color -p",
     "build": "NODE_ENV=production ./node_modules/webpack/bin/webpack.js --bail -p",
     "build:develop": "NODE_ENV=development ./node_modules/webpack/bin/webpack.js --bail --color -d",
     "test": "jest",
+    "jest-update-snapshot": "jest --updateSnapshot",
     "lint:scss": "stylelint src/scss",
-    "lint:scss:fix": "stylelint src/scss --fix"
+    "lint:scss:fix": "stylelint src/scss --fix",
+    "yalc-push": "nodemon -x 'yalc push'"
   },
   "dependencies": {
     "classnames": "^2.2.6",
     "date-fns": "^1.29.0",
     "formik": "^1.4.0",
     "moment-business-days": "^1.1.3",
+    "node-sass": "^4.11.0",
```

Comment:
Was it intentional to move`node-sass` from devDependencies to dependencies?

Reply:
Well spotted 👍